components/x-gift-article/src/ReferAFriend.jsx

```diff
@@ -18,7 +18,7 @@ export default ({ rafTitle, rafDescription, urls, actions }) => {
 						value={urls.raf}
 						className="x-gift-article__url-input"
 						readOnly
-						aria-label="Gift free subscription shareable link"
+						aria-label={rafTitle}
```

Comment:
Yes it would contain the title of the section and in this case is the default .
![image](https://user-images.githubusercontent.com/102036944/198227235-17093a5d-8782-4969-a6e0-3923a4b1929e.png)